### ISSUE TITLE
Issue with checkbox on change in listview

### ISSUE THREAD

**Issue by mosuzaur** [navasmdc/MaterialDesignLibrary#248]
I have a listview with items that contains a checkbox and if I set onClickListener on checkbox, it is fine but the onClick event is not trigered; If I set onCheckboxChanged listener (the method that is overriten by this library) and I click some checkboxes from the first part of the list and then I scrolled down, there are also checkboxes already checked, also some values are missed when I am scrolling.
I know that is a problem with onCheckboxChanged() in listview, but for a normal checkbox I was able to use onClickListener and all was fine, but for this library I can't use onClickListener.
How can I solve this problem?

**Comment by deepak28290** [comment]
Facing the same issue with onClickListener not getting fired in checkbox. Any luck with the resolution ?

**Comment by mosuzaur** [comment]
No. How works for you onCheckboxChange, because for me has a strange behavior?
